mypyc/test-data/irbuild-classes.test

```diff
@@ -2686,3 +2686,18 @@ L0:
     r6 = PyObject_VectorcallMethod(r3, r5, 9223372036854775812, 0)
     keep_alive r2, self, key, val
     return 1
+
+[case testInvalidMypycAttr]
+from mypy_extensions import mypyc_attr
+
+@mypyc_attr("allow_interpreted_subclasses", "invalid_arg")  # E: 'invalid_arg' is not a supported `mypyc_attr` \
+                                                            # N: supported keys: 'allow_interpreted_subclasses', 'free_list_len', 'native_class', 'serializable'
```

Comment:
style nit: we usually wrap names in double-quotes in errors/notes, so it should be "invalid_arg", "mypyc_attr", "allow_interpreted_subclasses", etc.

Reply:
fixed